core-bundle/src/Filesystem/Dbafs/DbafsInterface.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Filesystem\Dbafs;
+
+use Contao\CoreBundle\Filesystem\FilesystemItem;
+use Symfony\Component\Uid\Uuid;
+
+interface DbafsInterface
```

Comment:
Would love to have some better description of what `string $path` is. Is it relative? Does it have to be relative? How - as a developer - am I supposed to use it? :)

Reply:
It's all relative to the DBAFS root. So typically, `foo/bar` in a DBAFS registered to the prefix `files` will effectively target `files/foo/bar`. Prefixes are a concept of the `DbafsManager`, though, which I deliberatley not included in the `DbafsInterface`. 

Improved the descriptions in see 2c4b2fdc1212666d3f2ee7dc6dfd49ff5d038c5d